src/examples/learn_quant/grammar.py

```diff
@@ -0,0 +1,49 @@
+from ultk.language.grammar import Grammar, Rule
+
+class QuantifierGrammar(Grammar):
+    """This is a grammar class but for experiments involving quantifiers.
+
+    The class contains functions to dynamically generate rules for the grammar, such as providing it with a variable or defined number of integer primitives.
+
+    Args:
+        Grammar (Grammar): The grammar class to inherit from.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Add any additional initialization code here
+
+    from typing import Union, List
```

Comment:
Two things: (1) in general, I'm a fan of all imports should go up top :).  (2) in newest version of Python, you can just use `list`, so no need to import `List`, and also can use `|` for union, so don't need that one either